src/Mailer/Message/CitizenActionContactParticipantsMessage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Mailer\Message;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\EventRegistration;
+use Ramsey\Uuid\Uuid;
+
+final class CitizenActionContactParticipantsMessage extends Message
+{
+    /**
+     * @param EventRegistration[] $recipients
+     * @param Adherent            $organizer
+     * @param string              $content
+     *
+     * @return CitizenActionContactParticipantsMessage
+     */
+    public static function create(array $recipients, Adherent $organizer, string $content): self
+    {
+        $recipients = array_values($recipients);
+        $recipient = array_shift($recipients);
```

Comment:
I think it would be better to let the caller decide which recipient should be the 1st one. It's not clear from the outside what it is performed in here.

Reply:
I agree with you, this system is not very clear.
The abstract Message class handles this logic (waiting for a **main** recipient + an array of recipients), this could be refactored but out of scope imho